packages/plugin-dom-layout/test/DomLayout.test.ts

```diff
@@ -549,7 +549,6 @@ describe('DomLayout', () => {
                             '</jw-test>' +
                             '</jw-body>' +
                             '<jw-footer></jw-footer>' +
-                            '<t-dialog></t-dialog>' +
```

Comment:
Why ? Just wondering.

Reply:
@Gorash says it's because I added the `Dialog` plugin to `BasicEditor` and that somehow has the effect of not including `<t-dialog>` if there is none to display. He says it's normal. I didn't search further.